tests/test_resource_monitor.py

```
@@ -0,0 +1,372 @@
+        # Change thresholds
+        monitor.thresholds.cpu_warning = 1.0
+        health2 = monitor.check_resource_health()
+        assert health2 is not None
+
+    def test_metrics_accuracy(self):
+        """Test that metrics are reasonably accurate"""
+        monitor = ResourceMonitor()
+        metrics = monitor.get_current_metrics()
+
+        # Get system metrics independently
+        cpu = psutil.cpu_percent(interval=0)
```

Ruff (0.14.10)

Local variable `cpu` is assigned to but never used

Remove assignment to unused variable `cpu`

(F841)